### ISSUE TITLE
Migrate Android to Kolibri 16 alpha14

### ISSUE THREAD

**Issue by manuq** [endlessm/kolibri-installer-android#143]
Here is a draft PR: https://github.com/endlessm/kolibri-installer-android/pull/139

Currently the patch `0001-Break-up-DynamicWhiteNoise-code.patch` fails:

https://ci.endlessos.org/job/kolibri-installer-android-pr/433/console

```
# Apply kolibri patches
patch -d src/ -p1 < patches/0001-Add-track-progress-information-to-channelimport.patch
patching file kolibri/core/content/tasks.py
Hunk #1 succeeded at 194 (offset 42 lines).
patch -d src/ -p1 < patches/0001-Break-up-DynamicWhiteNoise-code.patch
patching file kolibri/utils/kolibri_whitenoise.py
Hunk #2 succeeded at 221 (offset 14 lines).
Hunk #3 succeeded at 347 (offset 14 lines).
Hunk #4 FAILED at 362.
1 out of 4 hunks FAILED -- saving rejects to file kolibri/utils/kolibri_whitenoise.py.rej
make: *** [Makefile:80: src/kolibri] Error 1
```

**Comment by dbnicholson** [comment]
Presumably from these [two](https://github.com/learningequality/kolibri/commit/b90282831fae6e025d361254a8e6e91e4fffd74b) [commits](https://github.com/learningequality/kolibri/commit/202f7be5f13abc5ec8df9eae6f72f2d4f2781f33) that were added in 0.16.0a13.

**Comment by dbnicholson** [comment]
I pulled the patch back into kolibri git and `git rebase` was able to DTRT (I think).

**Comment by manuq** [comment]
Closed by: #139

**Comment by erikos** [comment]
Anything left to do here anything we can/have to verify? @manuq
